os/booting-on-azure.md

```diff
@@ -1,14 +1,10 @@
 # Running CoreOS on Microsoft Azure
 
-## Choosing a Channel
+## Choosing a channel
```

Comment:
Sentence case fits the way these documents tend to use a large number of structural headings.It looks less quaint, and we'll save review time by not trying to figure out the many variations of Title Case (which we retain for titles).

(Watchers, see the [style guide, *Heading style*](https://github.com/coreos/docs/blob/master/STYLE.md#heading-style).)